packages/renderer/noteStyle.ts

```diff
@@ -79,9 +80,10 @@ export default function(theme: any, options: Options = null) {
 			font-size: ${formatCssSize(theme.noteViewerFontSize)};
 			color: ${theme.color};
 			word-wrap: break-word;
-			line-height: ${theme.lineHeight};
 			background-color: ${theme.backgroundColor};
 			font-family: ${fontFamily};
+			font: ${options.fontStyle};
+			line-height: ${theme.lineHeight};
```

Comment:
reordered the `line-height` to be under `font` because `font` by default has a `line-height: default;`, which we don't want. We want `line-height` to be as it should be (in our case `1.6em`)